test/TestServerProjects/multiple-inheritance/Generated/Models/Cat.cs

```diff
@@ -28,14 +28,8 @@ public Cat(string name) : base(name)
         /// <param name="likesMilk"></param>
         /// <param name="meows"></param>
         /// <param name="hisses"></param>
-        /// <exception cref="ArgumentNullException"> <paramref name="name"/> is null. </exception>
         internal Cat(string name, bool? likesMilk, bool? meows, bool? hisses) : base(name)
         {
-            if (name == null)
```

Comment:
Just curious: why does this validation go away?

Reply:
It is validated in the base constructor. Previously HLC model has been using last constructor in collection, which may be initialization constructor if initialization and serialization constructors have the same signature. Serialization constructors explicitly skip serialization in HLC. I can revert this part to minimize the change.

Reply:
No, it's fine, I was just curious.  Thanks!